custom_components/frigate/views.py

```diff
@@ -216,13 +216,26 @@ def _create_path(self, path: str, **kwargs: Any) -> str | None:
 
         camera = path.split("/")[0]
         if path.endswith("clip.mp4"):
-            return f"clips/{camera}-{event_id}.mp4"
+            return f"api/events/{event_id}/clip.mp4"
```

Comment:
Does this work today? If not, it will presumably break the integration clip playing for anyone using an older version of Frigate. In that case, it would be good to explicitly check the server version against the expected one and print  a warning. 

Example from a different integration: https://github.com/home-assistant/core/blob/dev/homeassistant/components/hyperion/__init__.py#L157

Reply:
Yes, we should add a version check in here.